aten/src/ATen/native_parse.py

```diff
@@ -227,29 +223,29 @@ def parse_arguments(args, func_variants, declaration, func_return):
     def check_topt_representation(topt_representation):
         for idx, supported_topt in enumerate(supported_topt_arguments):
             matches = True
-            matches = matches and topt_representation[0] == supported_topt[0]
-            matches = matches and topt_representation[1] == supported_topt[1]
-            matches = matches and topt_representation[2] == supported_topt[2]
+            for i, topt in enumerate(supported_topt):
+                matches = matches and topt_representation[i] == topt
```

Comment:
`matches = any(topt_representation[i] == topt for i, topt in enumerate(supported_topt))`